src/Components/ArticleByIdPageComponents/Votes.jsx

```diff
@@ -0,0 +1,10 @@
+const Votes =()=>{
+return(
+    <section id="votesButtons">
+    <button id="upvote">Upvote</button>
+    <button id="downvote">Downvote</button>
+    </section>
+)
```

Comment:
A little bit early to include this since it's not complete yet, our PRs shouldn't really have WIP features, since they'll be on their way to the main version of the app. Definitely something we will be doing on this project, but not quite yet!

Reply:
Thanks for the heads up, will bear this in mind for next tasks!